marketplace-elements.js

```diff
@@ -86,9 +86,15 @@
             },
             createdCallback: {
                 value: function () {
+                    var root = this;
```

Comment:
I switched these to use `function () {}.bind(this)` to be consistent. Left the classes I didn't touch alone.